backend/beyond_the_loop/models/completions.py

```
@@ -0,0 +1,90 @@
+        try:
+            with get_db() as db:
+                result = Completion(**completion.model_dump())
+                db.add(result)
+                db.commit()
+                db.refresh(result)
+                if result:
+                    return CompletionModel.model_validate(result)
+                else:
+                    print("insertion failed", result)
+                    return None
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)